jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -235,14 +235,6 @@ def _parse_image(app, file_obj, data_label, ext=None, parent=None):
         if sci_ext is None and data.ndim == 2 and ('[DATA' in data_label or '[SCI' in data_label):
             sci_ext = data_label
 
-        if isinstance(data.coords, GWCS) and (data.coords.bounding_box is not None):
-            # keep a copy of the original bounding box so we can detect
-            # when extrapolating beyond, but then remove the bounding box
-            # so that image layers are not cropped.
-            # NOTE: if extending this beyond GWCS, the mouseover logic
-            # for outside_*_bounding_box should also be updated.
-            data.coords._orig_bounding_box = data.coords.bounding_box
```

Comment:
I do not understand why this is removed. There is still an option for user to choose GWCS. It was specifically requested that we ignore the bounding box because mouseover get messed up when there are bounding boxes and user looking at mosaic. This is not just for Orientation.

Reply:
If (2) is acceptable, do you want to see that as part of this PR, or an immediate follow-up PR?

Reply:
I cannot speak for Roman use cases. Here is the original use case that required us to ignore the bounding box: User loads 4 JWST exposures (they have GWCS, not FITS WCS; but distortion is not an issue, just how parser grabbed WCS info). Let's forget about Orientation for a second, so first loaded image is the reference image. The exposures all get displayed at the same time because we support partial opacity. They have different bounding boxes. So when user mouse over to some parts outside one of them bounding boxes, it shows NaN. But user wants to see some real info because again, distortion is not problem here.

Roman support should not introduce regression to other non-Roman use cases. Or if it does, we have to ask stakeholders if they are okay with it.

Reply:
I think after offline discussions, we're in favor of keeping the changes as proposed, so I am marking this as resolved. Thanks for your time.